spec/lib/msf/base/sessions/command_shell_spec.rb

```diff
@@ -0,0 +1,159 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+COMMANDS = %i[help background sessions resource shell download upload source irb pry]
+
+RSpec.describe Msf::Sessions::CommandShell do
```

Comment:
It'd be nice to clean these up to align with some of the conventions from https://www.betterspecs.org/


this should follow the conventions of https://www.betterspecs.org/#describe

i.e. 

```
describe '.type' do
 # ...
end

describe '.can_cleanup_files' do
  # ...
end

# ... etc ...
```

And ensure `context` blocks follow the naming convention of `context 'when ...' do`

https://www.betterspecs.org/#contexts